integration/solana/debug_buffer.spec.ts

```diff
@@ -0,0 +1,146 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import { Connection, Keypair, PublicKey, sendAndConfirmTransaction, SystemProgram, Transaction } from '@solana/web3.js';
+import expect from 'expect';
+import { loadContract } from './setup';
+import { Program, Provider, BN } from '@project-serum/anchor';
+
+describe('DebugBuffer', function () {
+    this.timeout(150000);
+    
+    let program: Program;
+    let storage: Keypair;
+    let payer: Keypair;
+    let provider: Provider;
+
+    before(async function () {
+        ({ program, storage, payer, provider } = await loadContract('Printer'));
+    });
+
+    it('Prints runtime errors', async function () {
+  
+        try {
+            let res = await program.methods.setStorageBytes().accounts({ dataAccount: storage.publicKey }).simulate();
+        }
+        catch (e: any) {
+            const logs = e.simulationResponse.logs;
+            expect(logs).toContain('Program log: set storage index out of bounds in  file: 1, line: 42,10');
```

Comment:
The formatting can be better. We need the file name here. I think `filename:line-number:column-number` would be much better. WDYT @xermicus

Reply:
ACK